src/main/java/core/basesyntax/service/impl/FileReaderImpl.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+
+public class FileReaderImpl implements FileReader {
+    @Override
+    public List<String> read(String filePath) {
+        try {
+            return Files.readAllLines(Paths.get(filePath));
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read file by path: " + filePath, e);
+        }
+    }
+}

```

Comment:
It seems like you are not using try-with-resources or closing the resources explicitly. When working with I/O operations, it's important to ensure that resources are closed to prevent potential resource leaks. Consider using try-with-resources for automatic resource management.